src_base/xevd.c

```diff
@@ -1645,7 +1652,7 @@ void xevd_flush(XEVD_CTX * ctx)
 int xevd_dec_nalu(XEVD_CTX * ctx, XEVD_BITB * bitb, XEVD_STAT * stat)
 {
     XEVD_BSR  *bs = &ctx->bs;
-    XEVD_SPS  *sps = &ctx->sps;
+    XEVD_SPS  *sps = &ctx->sps_array[ctx->sps_id];
```

Comment:
is it safe to use ctx->sps_id without checking whether the ctx->sps_id is existed or not?

Reply:
Ah, ctx->sps_id is not actual syntax value. 
it seems to be current maximum sps id?

Reply:
the variable name seems to be confuzed.

Reply:
I think we need a assert after incrementing sps_id, not to go beyond 16